### PR TITLE
Website: Abort publish on PRs

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
It seems like $TRAVIS_BRANCH describes the branch to be merged in on Pull Requests, therefore we need to check if it is a PR individually.

And it still deploys on master ❤️ => https://travis-ci.org/DanielMSchmidt/detox/jobs/347647168

### CODE REVIEW

File: scripts/publish-website.sh [hunk 1]
```diff
@@ -2,14 +2,16 @@
 
 set -e
 
-if [[ "$TRAVIS_BRANCH" != "master" ]]; then 
-  echo 'Not deploying the website'; 
+BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
```
Comment (rotemmiz) [inline scripts/publish-website.sh]: TRAVIS_PULL_REQUEST_BRANCH can also be master branch on the forked library.
I think you may need to just check `if (!TRAVIS_PULL_REQUEST && TRAVIS_BRANCH==master)`

Comment (rotemmiz) [reply scripts/publish-website.sh]: Ping @danielmschmidt

Comment (DanielMSchmidt) [reply scripts/publish-website.sh]: Pong, will implement it somewhen this week, if you like to see it faster feel free to just push to the branch ;)

**Comment by rotemmiz** [comment]
Fixed by https://github.com/wix/detox/commit/7bc474209f37c519af777628e9c802c3c5648ba5

**Pull request closed without merging**